Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -64,9 +65,32 @@
 ]
 
 
+def get_location_of_reviewer(assigned_prs_per_potential_reviewer: dict) -> int:
```

Comment:
add to docstring what `key: value ` represents for `assigned_prs_per_potential_reviewer` dict